srcpkgs/coccinelle/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'coccinelle'
 pkgname=coccinelle
 version=1.0.8
-revision=5
+revision=6
 build_style=gnu-configure
```

Comment:
Is the revbump necessary, now that `ocaml-num` no longer changes?

Reply:
Oh indeed.